hms-app/src/components/card-details/VotingHandler.tsx

```diff
@@ -84,7 +78,8 @@ export default function VotingHandler(props: IProps) {
             autoClose: 2000,
           })
         } else {
-          check(true)
+          check(false)
```

Comment:
changing from `true` to `false` makes the button behave correctly (which took me a long time to realize 😌)